Sources/FindInPageIOSJSSupport/FindInPageIOSJSSupport.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+public struct FindInPageIOSJSSupport {
+    public static var Bundle: Bundle = .module
```

Comment:
Isn't the name of this var wrong?  Shouldn't it be:

```suggestion
    public static var bundle: Bundle = .module
```

Reply:
That's a good point, I used the content scope scripts one as a starting point which is wrong